agent/tool-scripts/perf

```diff
@@ -116,14 +116,15 @@ tool_output_dir=$tool_dir/$tool # all tools keep data in their tool specific dir
 tool_cmd_file="$tool_output_dir/$tool.cmd"
 tool_pid_file=$pbench_tmp/$group.$iteration.$tool.pid
 tool_output_file=$tool_output_dir/$tool.txt
+
+mkdir $tool_output_dir
```

Comment:
Are we sure we always want to do this?  It used to only occur on start.  Do we need this just to make unit tests pass?